app/server.go

```diff
@@ -112,20 +112,20 @@ type Server struct {
 
 	newStore func() store.Store
 
-	htmlTemplateWatcher            *utils.HTMLTemplateWatcher
-	sessionCache                   cache.Cache
-	seenPendingPostIdsCache        cache.Cache
-	statusCache                    cache.Cache
-	configListenerId               string
-	licenseListenerId              string
-	logListenerId                  string
-	clusterLeaderListenerId        string
-	searchConfigListenerId         string
-	searchLicenseListenerId        string
-	loggerMetricsLicenseListenerId string
-	configStore                    config.Store
-	asymmetricSigningKey           *ecdsa.PrivateKey
-	postActionCookieSecret         []byte
+	htmlTemplateWatcher     *utils.HTMLTemplateWatcher
+	sessionCache            cache.Cache
+	seenPendingPostIdsCache cache.Cache
+	statusCache             cache.Cache
+	configListenerId        string
+	licenseListenerId       string
+	logListenerId           string
+	clusterLeaderListenerId string
+	searchConfigListenerId  string
+	searchLicenseListenerId string
+	loggerLicenseListenerId string
```

Comment:
This seems unrelated?

Reply:
Previously this listenerid was just for logger metrics, but now we have multiple logger related action on a license change to faciliate removing unlicensed targets (see below).  This var was renamed to indicated it is for all logger related license watching.  The rest is just gofmt adjusting the column based on shorter name.